src/playlist-loader.js

```diff
@@ -249,8 +251,8 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     let currentBandwidth = loader.media().attributes.BANDWIDTH || 0;
 
     return !(loader.master.playlists.filter((playlist) => {
-      let enabled = typeof playlist.excludeUntil === 'undefined' ||
-                      playlist.excludeUntil <= Date.now();
```

Comment:
These two lines are the same thing as the lines 233-235. I think making these two lines a function that takes a playlist parameter would be better (something like `isPlaylistEnabled_(playlist) {...}`. Then the `enabledPlaylists_` function becomes `return loader.master.playlists.filter(isPlaylistEnabled_).length` and these two lines become just something like `let enabled = isPlaylistEnabled_(playlist);`